tests/unit/metric/LMNN_unittest.cc

```diff
@@ -62,6 +62,57 @@ TEST(LMNN,train_identity_init)
 	SG_UNREF(lmnn)
 }
 
+TEST(LMNN, train_termination)
+{
+	// create features, each column is a feature vector
```

Comment:
could you remove all these superfluous comments?

Reply:
also, does this test share code with other tests? If so it should be pulled out and shared

Reply:
code that creates features and labels could be shared with other tests, but this could make it difficult to modify test input. shall we do this refactor?